backend/plugins/sales_api/src/modules/sales/reports/graphql/resolvers/queries/reportsQueries.ts

```
@@ -0,0 +1,131 @@
+    return forecastRevenue(models, subdomain, filters);
+  },
+  dealsByStage: async (
+  _root: any,
+  { filters = {}, sort = "createdAt", limit = 100, skip = 0 }: 
+  { filters: any; sort?: string; limit?: number; skip?: number },
+  { models }: IContext,
+) => {
+  const match = buildFullMatch(filters);
+  
+  // Parse sort direction
+  let sortObj: any = {};
```

ESLint

[error] 'sortObj' is never reassigned. Use 'const' instead.

(prefer-const)